inc/spbc-backups.php

```diff
@@ -296,3 +296,17 @@ function spbc_backups_count_found()
 
     return $count ? key($count) : 0;
 }
+
+function spbc_file_has_backup($real_path)
+{
+    global $spbc, $wpdb;
+    $real_path = $spbc->is_windows ? str_replace('/', '\\', $real_path) : $real_path;
+    $query = 'SELECT * FROM ' . SPBC_TBL_BACKUPED_FILES;
+    $result = $wpdb->get_results($query, ARRAY_A);
```

Comment:
Could we immediately look for a specific file in the request? Without a foreach.

'SELECT * FROM ' . SPBC_TBL_BACKUPED_FILES WHERE real_path = $real_path
$wpdb->get_var or get_row

Reply:
No. Windows paths are incompatible with DB escaping on this iteration. I've spent a lot of time riding this out but had no success. So I decided to wrap this through php array.

Reply:
Probably, I will check this again on the next sprint during backups refactoring.